pontoon/contributors/static/css/settings.css

```diff
@@ -64,42 +60,98 @@
 }
 
 #main .field {
+  color: #aaaaaa;
+  font-size: 16px;
   margin-bottom: 20px;
   text-align: left;
 }
 
+#main .field label {
+  display: inline-block;
+  margin-right: 25px;
+  padding-top: 5px;
+  text-align: right;
+  vertical-align: top;
+  width: 295px;
+}
+
 #main .field input,
 #main .field textarea {
-  color: #ffffff;
+  color: #aaaaaa;
   background: #333941;
   border: 1px solid #4d5967;
   border-radius: 3px;
   float: none;
   padding: 4px;
   box-sizing: border-box;
+  width: 295px;
+}
+
+#main .field .toggle-button {
```

Comment:
I wanted to move the toggle-button CSS to a separate toggle.css file, but when I added it to the `PIPELINE_CSS`, the file didn't get included in the settings.css.

Do I need to do something special to re-build the file since the recent build refactor? It's been a while since I added a new CSS file in the `/pontoon` folder. I also didn't find anything relevant in the docs.

Reply:
During development, if you apply a change like this:
```diff
--- a/pontoon/settings/base.py
+++ b/pontoon/settings/base.py
@@ -359,6 +359,7 @@ PIPELINE_CSS = {
             "css/multiple_team_selector.css",
             "css/contributor.css",
             "css/team_selector.css",
+            "css/toggle-button.css",
             "css/settings.css",
         ),
         "output_filename": "css/settings.min.css",
```
you should be able to see a separate request for a `toggle-button.css`, rather than having its contents inlined into `settings.css`. It's only during the prod build that they're merged into `settings.min.css`.

Reply:
Hmmm, not sure why CSS didn't get applied the first time I tried. Now everything is fine. 

Sorry for the false alarm!